web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.test.tsx

```diff
@@ -662,7 +663,11 @@ describe("<AnyTimeActionTaxClearanceCertificateReviewElement />", () => {
     });
   });
 
-  it("renders the download page when the api post request is successful", async () => {
+  it("renders the page to download the certificate when the api post request is successful", async () => {
+    const pdfUrl = "blob:http://test-url";
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const mockCreateObjectURL = jest.fn((_) => pdfUrl); // `_` is mean to be unused
```

Comment:
[dust] There's an eslint rule for not giving the unused vars warning for parameters that start with underscore. Would you recommend us turning that on for our codebase?

Reply:
I know you care a lot about the developer experience on our team, so I'd love for you to bring it up as a 16th minute!